recipes/cvloop/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cvloop" %}
+{% set version = "0.3.2" %}
+{% set sha256 = "6c6e92420c99a938ffc271b0dba549814bdd13c5821e34eefe91f0bed469bc69" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - python 3.6.1
+    - opencv 3.2.0
+    - numpy 1.12.0
+    - matplotlib 2.0.0
+    - jupyter_core 4.3.0
+    - jupyter 1.0.0
```

Comment:
Since there is no build step, it's only install time. I did not know that it can do both. Also it might not even need jupyter, I will figure this out. So far I added all dependencies as the CIs failed. I will try fixing it!